lib/gitlab/client/projects.rb

```diff
@@ -471,5 +471,23 @@ def star_project(id)
     def unstar_project(id)
       delete("/projects/#{url_encode id}/star")
     end
+
+    # Get projects by user
+    #
```

Comment:
Could you add a link to the api doc like:
```
@see https://docs.gitlab.com/ee/api/projects.html#list-user-projects
```

Reply:
Done! Thanks :)